src/varity/ref_gene.clj

```diff
@@ -58,24 +58,30 @@
     (update m :cds-end-stat keyword)
     (update m :exon-frames parse-exon-pos)))
 
-(defn- load-ncbi-file
```

Comment:
Since we released the function, I think it would be better to keep it and mark it as deprecated for backward compatibility.

Reply:
@federkasten 
Thank you for the review. It sounds good but if this function was used from outside of this namespace,  that should be rewritten because it is defined as private. What do you think?

Reply:
Oops. Sorry, I missed that this is a private function.